sql/moz-fx-data-shared-prod/fenix_derived/attributable_clients_v1/init.sql

```diff
@@ -0,0 +1,115 @@
+CREATE TEMP FUNCTION sum_map_values(map ARRAY<STRUCT<key STRING, value INT64>>)
+RETURNS INT64 AS (
+  (SELECT SUM(value) FROM UNNEST(map))
+);
+
+CREATE OR REPLACE TABLE
+  fenix_derived.attributable_clients_v1
+PARTITION BY
+  submission_date
+CLUSTER BY
+  adjust_network,
+  adjust_campaign,
```

Comment:
we have almost 10k campaign values with installs since August, so that may not be the best clustering field. I'd leave it out and maybe add country.

Reply:
Even with a long tail it can still be useful. I'll be using this one for filtering so I'd rather keep it.

Reply:
how about adding country? We always use that one.

Reply:
Sure, we can add it in

Reply:
I'd suggest having country as the first or second field since the order matters (it won't leverage it unless the query is also filtering on campaign if country comes after it), but I'm good with it as it is 👍

Reply:
Usually, secondary/tertiary defined clustered fields will be used in filters or joins, but they just won't be as effective at filtering out partitions.

I think in general this table is going to be such an improvement on `marketing_attributable_metrics` that it won't matter - however if we end up with a higher need on fast queries for countries over campaigns, we can switch this up.